### ISSUE TITLE
picker scrolling speed & page scroll on browser

### ISSUE THREAD

**Issue by ShawkatAbuElnaser** [eliasab16/timeless-picker#3]
When scrolling the picker using the mouse wheel on the browser, the page itself also scrolls. The two scrolls interfere with each other. Also, the scrolling speed on the browser is not well adjusted like it is on mobile - the scrolling is way too fast.

**Comment by eliasab16** [comment]
Thanks for the feedback, will work on it soon.

I looked into the scrolling speed issue and for me the mouse wheel worked fine, but I experienced the same issue that you describe when using the touchpad. Were you using a touchpad or a mouse? 

As for the other issue, it seems like the component’s scrolling speed mechanism isn’t well adjusted for the fast input by the touchpad. I will try to work on this soon as well.

**Comment by eliasab16** [comment]
Thanks for the feedback, will work on it soon.

I looked into the scrolling speed issue and for me the mouse wheel worked fine, but I experienced the same issue that you describe when using the touchpad. Were you using a touchpad or a mouse?

As for the other issue, it seems like the component’s scrolling speed mechanism isn’t well adjusted for the fast input by the touchpad. I will try to work on this soon as well.

**Comment by ShawkatAbuElnaser** [comment]
I was using a touchpad. Thanks for the reply.

**Comment by tigrenok00** [comment]
@eliasab16 Hi, is there a change you could fix the speed factor when using touch (on mobile browsers)?

**Comment by eliasab16** [comment]
> @eliasab16 Hi, is there a change you could fix the speed factor when using touch (on mobile browsers)?

Yes. Sorry haven't had the time to work on this. Will do so soon.

**Comment by eliasab16** [comment]
@tigrenok00 is your issue that the picker isn't scrolling fast enough?
